tools/distrib/python/grpcio_tools/setup.py

```diff
@@ -152,14 +151,7 @@ def new_compile(obj, src, ext, cc_args, extra_postargs, pp_opts):
 
         self.compiler._compile = new_compile
 
-        try:
-            build_ext.build_ext.build_extensions(self)
-        except Exception as error:
```

Comment:
Can we keep this `except` and diagnose part?

Reply:
Usually we can get the build error (through stderr) regardless of this change and this code needs some change if we want to keep (e.g. class `CommandError` turned out to not exist in this module). Any value of having this one?

Reply:
@gnossen Look like we're only diagnose `AttributeError` and `CompileError`, `AttributeError` is mainly for older versions of setuptools which has been there for 8 years and doesn't seems to be relevant anymore.  Not so sure about `CompileError` though, do you think we should still keep those diagnoses?

Reply:
In practice, no one's reporting this error, so I think it's likely that it's never being hit. It's probably fine to remove this try-except block.